Paper: teaching machine learning in professional education department

nadiakap opened the pull request:
If you are creating this PR in order to submit a draft of your paper,
see http://procbuild.scipy.org/ for logs generated by the build
process.

See the [project readme](https://github.com/scipy-conference/scipy_proceedings#build-your-paper)
for more information.


*Editor*: <!--editor-->@deniederhut<!--end-editor-->

*Reviewers*: <!--reviewers-list-->@deniederhut<!--end-reviewers-list-->

nadiakap committed:
added my paper

nadiakap commented:
thanks

nadiakap commented:
I included bibliography into .tex file
is this acceptable?

nadiakap committed:
fix typos in section titles

nadiakap committed:
deleted bibliography section from metadata

deleted bibliography section from metadata

nadiakap committed:
add bib file

delete bibliography from .tex and add bib file

nadiakap committed:
add references

add references
fix typos

nadiakap commented:
I fixed couple of typos and the bibliography is in separate file , paper builds but still shows all the old typos, and it cannot find the bibliography (question marks instead of citing)
not sure what is wrong

deniederhut commented:
@scoobies assign @deniederhut as editor

scoobies commented:
Assigned! @deniederhut is now the editor

deniederhut commented:
@scoobies build status

scoobies commented:
{"nadiakap-2023"=>{"data"=>{"build_output"=>"[*] Updating proceedings build tools in {self.master_repo_path}...\nAlready up to date.\n[*] Check out paper repository...\nCloning into '/tmp/tmp6xcfwymo'...\nMoving proceedings build tools to temp directory...\n[*] Build the paper...\nBuilding: nu\n*** Warning: /tmp/tmp6xcfwymo/publisher/../output/nu/paper_stats.json does not exist.\n*** Warning: /tmp/tmp6xcfwymo/publisher/_build/toc.json does not exist.\n*** Warning: /tmp/tmp6xcfwymo/publisher/_build/other.json does not exist.\nError compiling BiBTeX\nbibtex error output:\n================================================================================\nb'This is BibTeX, Version 0.99d (TeX Live 2019/Debian)\\nThe top-level auxiliary file: paper.aux\\nThe style file: ieeetr.bst\\nDatabase file #1: mybib.bib\\nI was expecting a `,\\' or a `}\\'---line 38 of file mybib.bib\\n :   \\n :   doi = \"10.25080/majora-212e5952-019\"\\n(Error may have been on previous line)\\nI\\'m skipping whatever remains of this entry\\nI was expecting a `,\\' or a `}\\'---line 69 of file mybib.bib\\n :   \\n :   doi=\"10.1109/TIP.2021.3088611\"\\n(Error may have been on previous line)\\nI\\'m skipping whatever remains of this entry\\nRepeated entry---line 82 of file mybib.bib\\n : @article{MP\\n :            ,\\nI\\'m skipping whatever remains of this entry\\nI was expecting a `,\\' or a `}\\'---line 100 of file mybib.bib\\n :   \\n :   doi=\"10.1093/ajcn/nqx027\"\\n(Error may have been on previous line)\\nI\\'m skipping whatever remains of this entry\\nI was expecting a `,\\' or a `}\\'---line 111 of file mybib.bib\\n :   \\n :   doi=\"10.1007/BF00994018\"\\n(Error may have been on previous line)\\nI\\'m skipping whatever remains of this entry\\nWarning--I didn\\'t find a database entry for \"LAKLG\"\\nToo many commas in name 1 of \"Aizerman, M. A.; Braverman, Emmanuel M.; Rozoner, L. I.\" for entry ABR\\nwhile executing---line 969 of file ieeetr.bst\\nToo many commas in name 1 of \"Cortes, Corinna,Vapnik, Vladimir\" for entry VC\\nwhile executing---line 969 of file ieeetr.bst\\n(There were 7 error messages)\\n'\n================================================================================\n", "build_pdf_path"=>"/procbuild/procbuild/../cache/nadiakap-2023.pdf", "build_status"=>"success", "build_timestamp"=>"03/06 20:34"}, "status"=>"success"}}

deniederhut commented on papers/nu/mybib.bib:
```diff
@@ -0,0 +1,113 @@
+  journal = "Automation and Remote Control",
+  volume = 21,
+  issue = 3,
+  pages = {261-300}
```
@nadiakap the server isn't compiling the bibliography correctly because some of these entries are missing commas between the fields (e.g. there should be one here after `pages = {261-300}` and before `doi`

nadiakap replied:
@deniederhut, thank you

nadiakap committed:
fix bib

fix typos in bib file

nadiakap committed:
mybib update

add items to bib

nadiakap committed:
fix typos

fix typos in tex file

nadiakap committed:
add reference to food dataset

nadiakap committed:
fix mybib

nadiakap committed:
introduction fix

nadiakap committed:
add item to bibliography

add item to bibliography and its citing

nadiakap committed:
fix bib

fix names in bib

nadiakap committed:
fix mybib

fix errors in names and doi

nadiakap committed:
cite diet tutorial

add reference to diet problem  tutorial

deniederhut commented:
@scoobies add @shubhes to reviewers

scoobies commented:
@shubhes added to the reviewers list!

deniederhut commented:
@scoobies code of conduct

scoobies commented:
Our CoC: https://www.scipy2023.scipy.org/code-of-conduct

scoobies commented:
Reports of abusive or harassing behavior may be reported to scipy@enthought.com

nadiakap committed:
fix errors in mybib

fix mybib

nadiakap committed:
fix bib

deniederhut commented:
@scoobies add @ybouhara to reviewers

scoobies commented:
@ybouhara added to the reviewers list!

nadiakap committed:
add reference to neos

add reference and citation

nadiakap committed:
fix formatting

nadiakap committed:
fix neos citing

nadiakap committed:
fix url in bib

deniederhut commented:
Hey @shubhes and @ybouhara! Friendly reminder that your initial review is due June 23

nadiakap committed:
add to bib

nadiakap committed:
add another project description

nadiakap committed:
bibliography correction

nadiakap commented:
hello. how do I add to bibliography the reference to a paper in Russian?
will this  work:
@Article{KP1,
  author =  "A.I Kaplinskii and A.M Pesin",
  year =    "2018",
  title =   "Fast converging optimization methods for multi layer perceptrons",
  journal = "Vestnik, the Journal of Voronezh University",
  doi = "10.17308/sait.2018.4/1261"
  note = {in Russian. Каплинский, А. И., & Песин, А. М. (2018). О построении быстро сходящихся методов негладкой оптимизации многослойного персептрона. Вестник ВГУ. Серия: Системный анализ и информационные технологии, (4), 117-122.}
}

deniederhut commented:
I'm not sure the note will show up in the compiled paper but you can try!

nadiakap committed:
fix bib

nadiakap committed:
sub sub sub tiitle fix

nadiakap committed:
sub title fix

nadiakap committed:
sub title fix

nadiakap commented:
paper does not compile.. not sure what is wrong. how can i see the log ?

nadiakap committed:
reference fix

deniederhut commented:
@scoobies build status

scoobies commented:
{"nadiakap-2023"=>{"data"=>{"build_output"=>"[*] Updating proceedings build tools in {self.master_repo_path}...\nAlready up to date.\n[*] Check out paper repository...\nCloning into '/tmp/tmpx9f4mr0o'...\nMoving proceedings build tools to temp directory...\n[*] Build the paper...\nBuilding: nu\n*** Warning: /tmp/tmpx9f4mr0o/publisher/../output/nu/paper_stats.json does not exist.\n*** Warning: /tmp/tmpx9f4mr0o/publisher/_build/toc.json does not exist.\n*** Warning: /tmp/tmpx9f4mr0o/publisher/_build/other.json does not exist.\nError compiling BiBTeX\nbibtex error output:\n================================================================================\nb'This is BibTeX, Version 0.99d (TeX Live 2019/Debian)\\nThe top-level auxiliary file: paper.aux\\nThe style file: ieeetr.bst\\nDatabase file #1: mybib.bib\\nWarning--I didn\\'t find a database entry for \"YZWQ1.KP1\"\\nWarning--empty journal in LAKLG\\n(There were 2 warnings)\\nThis is pdfTeX, Version 3.14159265-2.6-1.40.20 (TeX Live 2019/Debian) (preloaded format=pdflatex)\\n restricted \\\\write18 enabled.\\nentering extended mode\\n(./paper.tex\\nLaTeX2e <2020-02-02> patch level 2\\nL3 programming layer <2020-02-14> (./IEEEtran.cls\\nDocument Class: IEEEtran 2015/08/26 V1.8b by Michael Shell\\n-- See the \"IEEEtran_HOWTO\" manual for usage information.\\n-- http://www.michaelshell.org/tex/ieeetran/\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/ot1ppl.fd)\\n-- Using IEEE Computer Society mode.\\n-- Using 8.5in x 11in (letter) paper.\\n-- Using PDF output.\\n-- This is a 10 point document.\\n) (/usr/share/texlive/texmf-dist/tex/latex/cmap/cmap.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/ifthen.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/fontenc.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/t1ppl.fd)<<t1.cmap>>)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/inputenc.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsmath.sty\\nFor additional information on amsmath, use the `?\\' option.\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amstext.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsgen.sty))\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsbsy.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsopn.sty))\\n(/usr/share/texlive/texmf-dist/tex/latex/booktabs/booktabs.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/color.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics-cfg/color.cfg)\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics-def/pdftex.def))\\n(/usr/share/texlive/texmf-dist/tex/latex/caption/caption.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/caption/caption3.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/keyval.sty))\\n\\nPackage caption Warning: Unknown document class (or package),\\n(caption)                standard defaults will be used.\\nSee the caption package documentation for explanation.\\n\\n) (/usr/share/texlive/texmf-dist/tex/latex/float/float.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/tools/longtable.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/caption/ltcaption.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/tools/array.sty) (./scipy.sty\\n(./status.sty (./draftwatermark.sty (./everypage.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/graphicx.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/graphics.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/trig.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics-cfg/graphics.cfg)))))\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/mathptmx.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/helvet.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/courier.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/amsfonts/amsfonts.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/amsfonts/amssymb.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/tools/bm.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/hyperref/hyperref.sty\\n(/usr/share/texlive/texmf-dist/tex/generic/ltxcmds/ltxcmds.sty)\\n(/usr/share/texlive/texmf-dist/tex/generic/iftex/iftex.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/pdftexcmds/pdftexcmds.sty\\n(/usr/share/texlive/texmf-dist/tex/generic/infwarerr/infwarerr.sty))\\n(/usr/share/texlive/texmf-dist/tex/generic/kvsetkeys/kvsetkeys.sty)\\n(/usr/share/texlive/texmf-dist/tex/generic/kvdefinekeys/kvdefinekeys.sty)\\n(/usr/share/texlive/texmf-dist/tex/generic/pdfescape/pdfescape.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/hycolor/hycolor.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/letltxmacro/letltxmacro.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/auxhook/auxhook.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/kvoptions/kvoptions.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/hyperref/pd1enc.def)\\n(/usr/share/texlive/texmf-dist/tex/generic/intcalc/intcalc.sty)\\n(/usr/share/texlive/texmf-dist/tex/generic/etexcmds/etexcmds.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/url/url.sty)\\n(/usr/share/texlive/texmf-dist/tex/generic/bitset/bitset.sty\\n(/usr/share/texlive/texmf-dist/tex/generic/bigintcalc/bigintcalc.sty))\\n(/usr/share/texlive/texmf-dist/tex/generic/atbegshi/atbegshi.sty))\\n(/usr/share/texlive/texmf-dist/tex/latex/hyperref/hpdftex.def\\n(/usr/share/texlive/texmf-dist/tex/latex/atveryend/atveryend.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/rerunfilecheck/rerunfilecheck.sty\\n(/usr/share/texlive/texmf-dist/tex/generic/uniquecounter/uniquecounter.sty)))\\n\\nPackage hyperref Warning: Option `pdftex\\' has already been used,\\n(hyperref)                setting the option has no effect on input line 43.\\n\\n(/usr/share/texlive/texmf-dist/tex/latex/multirow/multirow.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/textcomp.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/upquote/upquote.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/fancyvrb/fancyvrb.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/algorithms/algorithm.sty))\\n(/usr/share/texlive/texmf-dist/tex/latex/algorithmicx/algpseudocode.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/algorithmicx/algorithmicx.sty\\nDocument Style algorithmicx 1.2 - a greatly improved `algorithmic\\' style\\n)\\nDocument Style - pseudocode environments for use with the `algorithmicx\\' style\\n) (/usr/share/texlive/texmf-dist/tex/latex/l3backend/l3backend-pdfmode.def)\\n(./paper.aux) (/usr/share/texlive/texmf-dist/tex/latex/psnfss/t1ptm.fd)\\n-- Lines per column: 61 (exact).\\n(/usr/share/texlive/texmf-dist/tex/context/base/mkii/supp-pdf.mkii\\n[Loading MPS to PDF converter (version 2006.09.02).]\\n) (/usr/share/texlive/texmf-dist/tex/latex/epstopdf-pkg/epstopdf-base.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/latexconfig/epstopdf-sys.cfg))\\n(/usr/share/texlive/texmf-dist/tex/latex/hyperref/nameref.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/refcount/refcount.sty)\\n(/usr/share/texlive/texmf-dist/tex/generic/gettitlestring/gettitlestring.sty))\\n(./paper.out) (./paper.out) (./page_numbers.tex)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/t1phv.fd)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/ot1ztmcm.fd)<<ot1.cmap>>\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/omlztmcm.fd)<<oml.cmap>>\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/omsztmcm.fd)<<oms.cmap>>\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/omxztmcm.fd)<<omx.cmap>>\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/ot1ptm.fd)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/upzd.fd)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/ts1ptm.fd)\\n\\nLaTeX Warning: Citation `MP\\' on page 1 undefined on input line 235.\\n\\n[1{/var/lib/texmf/fonts/map/pdftex/updmap/pdftex.map}]\\n\\nLaTeX Warning: Citation `neos\\' on page 2 undefined on input line 258.\\n\\n\\nLaTeX Warning: Citation `ABR\\' on page 2 undefined on input line 273.\\n\\n\\nLaTeX Warning: Citation `CS\\' on page 2 undefined on input line 275.\\n\\n[2]\\n\\nLaTeX Warning: Citation `VC\\' on page 3 undefined on input line 297.\\n\\n\\nLaTeX Warning: Citation `RV\\' on page 3 undefined on input line 305.\\n\\n\\nUnderfull \\\\hbox (badness 1472) in paragraph at lines 402--403\\n[]\\\\T1/ptm/m/n/9.53561 Note that lin-ear com-bi-na-tion of in-de-pen-dent (as-su\\nmed)\\n\\nUnderfull \\\\hbox (badness 4995) in paragraph at lines 402--403\\n\\\\T1/ptm/m/n/9.53561 nor-mally dis-tributed ran-dom vari-ables $[][] []$ is nor-\\n\\n[3] (/usr/share/texlive/texmf-dist/tex/latex/psnfss/t1pcr.fd)\\n\\nLaTeX Warning: Citation `diet\\' on page 4 undefined on input line 420.\\n\\n\\nLaTeX Warning: Citation `MP\\' on page 4 undefined on input line 437.\\n\\n\\nLaTeX Warning: Citation `KAP1\\' on page 4 undefined on input line 437.\\n\\n\\nLaTeX Warning: Citation `KAP2\\' on page 4 undefined on input line 437.\\n\\n\\nLaTeX Warning: Citation `KAP3\\' on page 4 undefined on input line 437.\\n\\n\\nLaTeX Warning: Citation `LAKLG\\' on page 4 undefined on input line 437.\\n\\n\\nLaTeX Warning: Citation `RGMN\\' on page 4 undefined on input line 437.\\n\\n\\nLaTeX Warning: Citation `SGFLZ\\' on page 4 undefined on input line 437.\\n\\n\\nLaTeX Warning: Citation `ABR\\' on page 4 undefined on input line 447.\\n\\n\\nLaTeX Warning: Citation `MP\\' on page 4 undefined on input line 447.\\n\\n\\nLaTeX Warning: Citation `YZWQ\\' on page 4 undefined on input line 447.\\n\\n\\nLaTeX Warning: Citation `YZWQ1.KP1\\' on page 4 undefined on input line 447.\\n\\n(./paper.bbl\\n\\n! Package inputenc Error: Unicode character ^^B (U+0002)\\n(inputenc)                not set up for use with LaTeX.\\n\\nSee the inputenc package documentation for explanation.\\nType  H <return>  for immediate help.\\n ...                                              \\n                                                  \\nl.65   nested neu^^B\\n                    ral network for adaptive compression and quantization,\\'\\'...\\n!  ==> Fatal error occurred, no output PDF file produced!\\nTranscript written on paper.log.\\n'\n================================================================================\n[X] Paper build failed.\nA build error occurred during the self.retrieve_pdf step.", "build_pdf_path"=>"", "build_status"=>"Build failed, no pdf can be found", "build_timestamp"=>"29/06 21:02"}, "status"=>"fail"}}

deniederhut commented:
☹️ looks like bibtex is tripping over the unicode

deniederhut commented:
looks like you would need to add any unicode chars not already in latex by hand https://tex.stackexchange.com/a/107353

nadiakap committed:
fix reference

nadiakap commented:
looks like the error was in citing two sources  where they were separated by dot instead of comma. this is fixed

nadiakap commented:
deniederhut: for unicode chars - do you mean Russian letters or smth else?

nadiakap committed:
delete reference

nadiakap committed:
bold font

deniederhut commented:
I assumed it was the Russian characters -- did you look into which change caused the issues?

Also, neither of the two reviewers who agreed to review this paper have followed up, so I will be stepping in.

deniederhut commented:
@scoobies remove @ybouhara from reviewers

scoobies commented:
@ybouhara removed from the reviewers list!

deniederhut commented:
@scoobies remove @shubhes from reviewers

scoobies commented:
@shubhes removed from the reviewers list!

deniederhut commented:
@scoobies add @deniederhut  to reviewers

scoobies commented:
I'm sorry human, I don't understand that. You can see what commands I support by typing:

`@scoobies help`

deniederhut commented:
@scoobies help

scoobies commented:
Hello @deniederhut, here are the things you can ask me to do:

```

# List all available commands
@scoobies help

# Show our community Code of Conduct and Guidelines
@scoobies code of conduct

# Add to this issue's reviewers list
@scoobies add @username to reviewers

# Remove from this issue's reviewers list
@scoobies remove @username from reviewers

# Assign a user as the editor of this submission
@scoobies assign @username as editor

# Remove the editor assigned to this submission
@scoobies remove editor

# Add a user to this issue's assignees list
@scoobies add assignee: @username

# Remove a user from this issue's assignees list
@scoobies remove assignee: @username

# Builds paper
@scoobies build paper

# Checks build status
@scoobies build status

# Check the references of the paper for missing DOIs
# Optionally, it can be run on a non-default branch 
@scoobies check references
@scoobies check references from branch custom-branch-name

# Label issue with: paper
@scoobies mark paper

# Label issue with: needs-review. Remove labels: unready, needs-more-review, pending-comment, ready
@scoobies mark needs review

# Label issue with: needs-more-review. Remove labels: unready, needs-review, pending-comment, ready
@scoobies mark needs more review

# Label issue with: pending-comment. Remove labels: unready, needs-review, needs-more-review, ready
@scoobies mark pending comment

# Label issue with: ready. Remove labels: unready, needs-review, needs-more-review, pending-comment
@scoobies mark ready

# Label issue with: unready. Remove labels: needs-review, needs-more-review, pending-comment, ready
@scoobies mark not ready

# Label issue with: does_not_build:server
@scoobies mark server fail

# Remove labels: does_not_build:server
@scoobies mark server success

```

deniederhut commented:
@scoobies add @deniederhut to reviewers

scoobies commented:
@deniederhut added to the reviewers list!

nadiakap committed:
add project

deniederhut commented on papers/nu/metadata.yaml:
```diff
@@ -0,0 +1,19 @@
+---
+abstract:
+   - We suggest the curriculum for teaching machine learning in professional education department. Its special goal is to build deeper understanding of machine learning concepts and provide background for self study. 
```
I think what you mean to say here is "We propose a new curriculum..." or "We have created a differentiated curriculum...". In English, "suggest" means to raise for discussion something previously known.

deniederhut replied:
"in **a** professional education department"

deniederhut replied:
The way that I typically hear this described is in terms of "building intuition".

deniederhut replied:
Taking a step back -- the typical structure for an abstract is the following:

1. Historically, the field has known X.
2. X leaves Y unadressed.
3. We conducted practice Z to address Y.
4. From practicing Z, we learned A, B, and C, which may be applicable to other programs.

There is a lot of scholarship on teaching. There is a lot of scholarship on teaching machine learning! Andrew Ng has one opinion, and Jeremy Howard has a different opinion. They are both wildly intelligent people! This means you have room to step in with your contextualized experience and say something meaningful for your field! But for your experience to be understandable to other people who care about the same things that you care about, it needs to be framed in terms of previous work in education and machine learning. The audience you want to speak to will know this work!

deniederhut commented on papers/nu/metadata.yaml:
```diff
@@ -0,0 +1,19 @@
+---
+abstract:
+   - We suggest the curriculum for teaching machine learning in professional education department. Its special goal is to build deeper understanding of machine learning concepts and provide background for self study. 
+   - Other features include short length, project-based approach, and minimal required math and programming. However, student’s special skills such as advanced math, engineering or natural/social science  background play an important role when selecting individual educational projects. 
```
I think the framing here is important. The language suggests that the short length is a negative, counterposed with the special skills, which are a positive. But these are all positives! Busy people can't spend a year doing full time ML education. Our best people are busy because they are good at what they do! 

Another way to say this might be "we constructed a program that built on the existing talents of our cohorts without taking them away from areas where their skills were in great demand". It's hard to teach people meaningful things quickly! Celebrate that you did this!

deniederhut commented on papers/nu/paper_n_udler.tex:
```diff
@@ -0,0 +1,256 @@
+
+\subsection{Introduction \label{introduction}}
+
+The focus of the class is on basic machine learning (ML) ideas and concepts. This approach allows students to easily acquire the terminology and continue to study independently after finishing the class -  through books, videos  and software documentation.  The course is project-based. Each time the selection of the projects is different. It depends on the math knowledge of the students. First se ...
```
Same as my note from the abstract -- the framing we want here is that the field used to look like X. Y was missing, so we did Z. What are X, Y, and Z?

deniederhut commented on papers/nu/paper_n_udler.tex:
```diff
@@ -0,0 +1,256 @@
+
+\subsection{Objectives and Outcomes \label{objectives and outcomes}}
+
+The objective is to give a student enough skills to recognize ML problems around us, continue self-study using online materials, video lectures,  scientific papers,  write professional resume, find job or apply for advanced education program.
```
Okay so reading through this paper, I'm confused about who you think your audience is. Who will read this? What do you want them to learn? 

Here is a heuristic for you to use -- what did you think this project would look like when you started? What were you wrong about? What did you have to change while the project was ongoing? These are all things that might be useful to someone trying to do the same thing that you did.

nadiakap replied:
I see what may be confusing.. When thinking about professional education department we think of people in the middle of their career, and yes, most of the students are. But some students are college students ( engineering departments, business students, etc). They are taking ML class to see if this may help to give a different focus to their selected field of study.

may be I will describe these different types of students separately?

deniederhut replied:
Well, if the paper is about how to structure a machine learning course, then would the audience be other educators? If that's true, what would you want them to know? What do you wish you had known before you got started? What worked really well when you tried it? What didn't go the way you had planned?

I think talking about different kinds of students is fine -- classes are always heterogeneous!

deniederhut commented on papers/nu/paper_n_udler.tex:
```diff
@@ -0,0 +1,256 @@
+
+\subsubsection{Definition \label{definition}}
+
+Demonstration projects show how to solve real life problem using machine learning approach and using ML software libraries or other scientific software libraries. Almost half of class time is devoted to demonstration projects. Students projects are structured in the same manner as demonstration projects. After watching several demonstration projects students are able to start working on their own ...
```
What would Greg Wilson say about this?

deniederhut commented on papers/nu/paper_n_udler.tex:
```diff
@@ -0,0 +1,256 @@
+
+2. Creating mathematical model of the problem.
+
+A special focus is on representing the model in a form consistent with API of a software that is used to solve the problem.  We encourage the use of standard software tools to reduce the development time, increase the reliability of the solution, and require less math and programming skills.
```
Cite scipy and scikit-learn plese!

deniederhut commented on papers/nu/paper_n_udler.tex:
```diff
@@ -0,0 +1,256 @@
+
+Project example for a group with good programming skills, college-level mathematics  and special interest in optimization may be Applying proximal operator as a step in generalized global optimization algorithm derived from potential theory. See Appendix for the description.
+
+\subsection{Appendix \label{appendix}}
```
The appendix is longer than the paper. This feels backwards to me.

nadiakap replied:
deniederhut, thank you so much for the review!

nadiakap replied:
going to ask a couple of questions.. 
About the appending being longer than a paper: I'd lile to demonstrate one project from the beginning to the  end. Should it not be in the appendix?
I  guess it should be in demonstration projects section

deniederhut replied:
Yeah I think it's fine to have a case study in the main body of the paper.

deniederhut reviewed (changes requested):
Helping people get a good intuition about machine learning, how it works, and what its affordances are, is a crucial skill set in the twenty first century. This paper appears to address this topic, but neither engages with prior work in this area, nor provides enough detail for future practitioners to guide their own approach based on the tradeoffs, mid-course corrections, and ultimate learnings from the authors. 

What would improve this submission are the following:

1. a clear structure, which engages with what is already known, and either supports or challenges this; and,
2. the details about the tradeoffs that the organizers considered when designing their curriculum; and,
3. the implementation that the authors actioned; and,
4. any failures of that implementation, and especially any mid-plan corrections that they made, what precipitated these, and what the results of the intervention were

deniederhut commented:
@scoobies mark pending comment

nadiakap commented:
when should the corrected version be submitted?

nadiakap commented:
> I assumed it was the Russian characters -- did you look into which change caused the issues?
> 
according to the log the issue was in citing two sources and using dot instead of comma to separate them: instead of \cite{YZWQ1,KP1} it was \cite{YZWQ1.KP1}

this caused the error :this is YZWQ1.KP1\' on page 4 undefined

changed dot to comma and also removed all Russian letters from the bibliography.  it should compile now

> Also, neither of the two reviewers who agreed to review this paper have followed up, so I will be stepping in.

deniederhut commented:
@scoobies build status

scoobies commented:
{"nadiakap-2023"=>{"data"=>{"build_output"=>"[*] Updating proceedings build tools in {self.master_repo_path}...\nAlready up to date.\n[*] Check out paper repository...\nCloning into '/tmp/tmp25dl6bs7'...\nMoving proceedings build tools to temp directory...\n[*] Build the paper...\nBuilding: nu\n*** Warning: /tmp/tmp25dl6bs7/publisher/../output/nu/paper_stats.json does not exist.\n*** Warning: /tmp/tmp25dl6bs7/publisher/_build/toc.json does not exist.\n*** Warning: /tmp/tmp25dl6bs7/publisher/_build/other.json does not exist.\nError compiling BiBTeX\nbibtex error output:\n================================================================================\nb'This is BibTeX, Version 0.99d (TeX Live 2019/Debian)\\nThe top-level auxiliary file: paper.aux\\nThe style file: ieeetr.bst\\nDatabase file #1: mybib.bib\\nWarning--empty journal in LAKLG\\n(There was 1 warning)\\nThis is pdfTeX, Version 3.14159265-2.6-1.40.20 (TeX Live 2019/Debian) (preloaded format=pdflatex)\\n restricted \\\\write18 enabled.\\nentering extended mode\\n(./paper.tex\\nLaTeX2e <2020-02-02> patch level 2\\nL3 programming layer <2020-02-14> (./IEEEtran.cls\\nDocument Class: IEEEtran 2015/08/26 V1.8b by Michael Shell\\n-- See the \"IEEEtran_HOWTO\" manual for usage information.\\n-- http://www.michaelshell.org/tex/ieeetran/\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/ot1ppl.fd)\\n-- Using IEEE Computer Society mode.\\n-- Using 8.5in x 11in (letter) paper.\\n-- Using PDF output.\\n-- This is a 10 point document.\\n) (/usr/share/texlive/texmf-dist/tex/latex/cmap/cmap.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/ifthen.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/fontenc.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/t1ppl.fd)<<t1.cmap>>)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/inputenc.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsmath.sty\\nFor additional information on amsmath, use the `?\\' option.\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amstext.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsgen.sty))\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsbsy.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsopn.sty))\\n(/usr/share/texlive/texmf-dist/tex/latex/booktabs/booktabs.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/color.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics-cfg/color.cfg)\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics-def/pdftex.def))\\n(/usr/share/texlive/texmf-dist/tex/latex/caption/caption.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/caption/caption3.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/keyval.sty))\\n\\nPackage caption Warning: Unknown document class (or package),\\n(caption)                standard defaults will be used.\\nSee the caption package documentation for explanation.\\n\\n) (/usr/share/texlive/texmf-dist/tex/latex/float/float.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/tools/longtable.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/caption/ltcaption.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/tools/array.sty) (./scipy.sty\\n(./status.sty (./draftwatermark.sty (./everypage.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/graphicx.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/graphics.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics/trig.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/graphics-cfg/graphics.cfg)))))\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/mathptmx.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/helvet.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/courier.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/amsfonts/amsfonts.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/amsfonts/amssymb.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/tools/bm.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/hyperref/hyperref.sty\\n(/usr/share/texlive/texmf-dist/tex/generic/ltxcmds/ltxcmds.sty)\\n(/usr/share/texlive/texmf-dist/tex/generic/iftex/iftex.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/pdftexcmds/pdftexcmds.sty\\n(/usr/share/texlive/texmf-dist/tex/generic/infwarerr/infwarerr.sty))\\n(/usr/share/texlive/texmf-dist/tex/generic/kvsetkeys/kvsetkeys.sty)\\n(/usr/share/texlive/texmf-dist/tex/generic/kvdefinekeys/kvdefinekeys.sty)\\n(/usr/share/texlive/texmf-dist/tex/generic/pdfescape/pdfescape.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/hycolor/hycolor.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/letltxmacro/letltxmacro.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/auxhook/auxhook.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/kvoptions/kvoptions.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/hyperref/pd1enc.def)\\n(/usr/share/texlive/texmf-dist/tex/generic/intcalc/intcalc.sty)\\n(/usr/share/texlive/texmf-dist/tex/generic/etexcmds/etexcmds.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/url/url.sty)\\n(/usr/share/texlive/texmf-dist/tex/generic/bitset/bitset.sty\\n(/usr/share/texlive/texmf-dist/tex/generic/bigintcalc/bigintcalc.sty))\\n(/usr/share/texlive/texmf-dist/tex/generic/atbegshi/atbegshi.sty))\\n(/usr/share/texlive/texmf-dist/tex/latex/hyperref/hpdftex.def\\n(/usr/share/texlive/texmf-dist/tex/latex/atveryend/atveryend.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/rerunfilecheck/rerunfilecheck.sty\\n(/usr/share/texlive/texmf-dist/tex/generic/uniquecounter/uniquecounter.sty)))\\n\\nPackage hyperref Warning: Option `pdftex\\' has already been used,\\n(hyperref)                setting the option has no effect on input line 43.\\n\\n(/usr/share/texlive/texmf-dist/tex/latex/multirow/multirow.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/base/textcomp.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/upquote/upquote.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/fancyvrb/fancyvrb.sty)\\n(/usr/share/texlive/texmf-dist/tex/latex/algorithms/algorithm.sty))\\n(/usr/share/texlive/texmf-dist/tex/latex/algorithmicx/algpseudocode.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/algorithmicx/algorithmicx.sty\\nDocument Style algorithmicx 1.2 - a greatly improved `algorithmic\\' style\\n)\\nDocument Style - pseudocode environments for use with the `algorithmicx\\' style\\n) (/usr/share/texlive/texmf-dist/tex/latex/l3backend/l3backend-pdfmode.def)\\n(./paper.aux) (/usr/share/texlive/texmf-dist/tex/latex/psnfss/t1ptm.fd)\\n-- Lines per column: 61 (exact).\\n(/usr/share/texlive/texmf-dist/tex/context/base/mkii/supp-pdf.mkii\\n[Loading MPS to PDF converter (version 2006.09.02).]\\n) (/usr/share/texlive/texmf-dist/tex/latex/epstopdf-pkg/epstopdf-base.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/latexconfig/epstopdf-sys.cfg))\\n(/usr/share/texlive/texmf-dist/tex/latex/hyperref/nameref.sty\\n(/usr/share/texlive/texmf-dist/tex/latex/refcount/refcount.sty)\\n(/usr/share/texlive/texmf-dist/tex/generic/gettitlestring/gettitlestring.sty))\\n(./paper.out) (./paper.out) (./page_numbers.tex)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/t1phv.fd)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/ot1ztmcm.fd)<<ot1.cmap>>\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/omlztmcm.fd)<<oml.cmap>>\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/omsztmcm.fd)<<oms.cmap>>\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/omxztmcm.fd)<<omx.cmap>>\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/ot1ptm.fd)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/upzd.fd)\\n(/usr/share/texlive/texmf-dist/tex/latex/psnfss/ts1ptm.fd)\\n\\nLaTeX Warning: Citation `MP\\' on page 1 undefined on input line 235.\\n\\n[1{/var/lib/texmf/fonts/map/pdftex/updmap/pdftex.map}]\\n\\nLaTeX Warning: Citation `neos\\' on page 2 undefined on input line 258.\\n\\n\\nLaTeX Warning: Citation `ABR\\' on page 2 undefined on input line 273.\\n\\n\\nLaTeX Warning: Citation `CS\\' on page 2 undefined on input line 275.\\n\\n[2]\\n\\nLaTeX Warning: Citation `VC\\' on page 3 undefined on input line 297.\\n\\n\\nLaTeX Warning: Citation `RV\\' on page 3 undefined on input line 305.\\n\\n\\nUnderfull \\\\hbox (badness 1472) in paragraph at lines 402--403\\n[]\\\\T1/ptm/m/n/9.53561 Note that lin-ear com-bi-na-tion of in-de-pen-dent (as-su\\nmed)\\n\\nUnderfull \\\\hbox (badness 4995) in paragraph at lines 402--403\\n\\\\T1/ptm/m/n/9.53561 nor-mally dis-tributed ran-dom vari-ables $[][] []$ is nor-\\n\\n[3] (/usr/share/texlive/texmf-dist/tex/latex/psnfss/t1pcr.fd)\\n\\nLaTeX Warning: Citation `diet\\' on page 4 undefined on input line 420.\\n\\n\\nLaTeX Warning: Citation `MP\\' on page 4 undefined on input line 437.\\n\\n\\nLaTeX Warning: Citation `KAP1\\' on page 4 undefined on input line 437.\\n\\n\\nLaTeX Warning: Citation `KAP2\\' on page 4 undefined on input line 437.\\n\\n\\nLaTeX Warning: Citation `KAP3\\' on page 4 undefined on input line 437.\\n\\n\\nLaTeX Warning: Citation `LAKLG\\' on page 4 undefined on input line 437.\\n\\n\\nLaTeX Warning: Citation `RGMN\\' on page 4 undefined on input line 437.\\n\\n\\nLaTeX Warning: Citation `SGFLZ\\' on page 4 undefined on input line 437.\\n\\n\\nLaTeX Warning: Citation `ABR\\' on page 4 undefined on input line 447.\\n\\n\\nLaTeX Warning: Citation `MP\\' on page 4 undefined on input line 447.\\n\\n\\nLaTeX Warning: Citation `YZWQ\\' on page 4 undefined on input line 447.\\n\\n\\nLaTeX Warning: Citation `YZWQ1\\' on page 4 undefined on input line 447.\\n\\n\\nLaTeX Warning: Citation `KP1\\' on page 4 undefined on input line 447.\\n\\n(./paper.bbl\\n\\n! Package inputenc Error: Unicode character ^^B (U+0002)\\n(inputenc)                not set up for use with LaTeX.\\n\\nSee the inputenc package documentation for explanation.\\nType  H <return>  for immediate help.\\n ...                                              \\n                                                  \\nl.65   nested neu^^B\\n                    ral network for adaptive compression and quantization,\\'\\'...\\n!  ==> Fatal error occurred, no output PDF file produced!\\nTranscript written on paper.log.\\n'\n================================================================================\n[X] Paper build failed.\nA build error occurred during the self.retrieve_pdf step.", "build_pdf_path"=>"", "build_status"=>"Build failed, no pdf can be found", "build_timestamp"=>"09/07 22:13"}, "status"=>"fail"}}

deniederhut commented on papers/nu/mybib.bib:
```diff
@@ -0,0 +1,150 @@
+  doi = "10.1007/978-3-540-48125-6"
+}
+
+@Article{YZWQ,
```
The build is currently failing because there are two https://unicodeplus.com/U+0002 in this citation. It's unfortunate that github is not showing it -- here's what it looks like when I open your bibfile in vscode:

```
@Article{YZWQ,
  author =  "Yufei Cui and Ziquan Liu and Wuguannan Yao and Qiao Li and Antoni B. Chan and Tei-wei Kuo and Chun Jason Xue",
  year =    "2020",
  title =   "Fully nested neural network for adaptive compression and quantization",
  journal = "Proceedings of the Twenty-Ninth International Joint Conference on Artificial Intelligence",
  doi = "10.24963/ijcai.2020/288"
}
```

It might be helpful to open this in a plaintext editor to make sure there are no other weird characters floating around.

deniederhut commented:
> when should the corrected version be submitted?


Final author revision deadline is the 14th https://github.com/scipy-conference/scipy_proceedings#author-deadlines

deniederhut commented:
@nadiakap we're getting pretty close to the deadline here. How are the updates coming along?

nadiakap commented:
> @nadiakap we're getting pretty close to the deadline here. How are the updates coming along?

this is the updated abstract:

Use of ML has become mainstream in many areas of business. There is no shortage of courses offering different paths to mastering ML skills. The most well-known approaches are those of Andrew Ng and Jeremy Howard. Andrew Ng's approach assumes a decent mathematical background and is essentially deductive, while Jeremy Howard's approach is more inductive and requires high school level of math and limited coding skills. Obviously, both are valuable in their own ways.
In the practice of continual education, however, the unique challenge is to work with people of diverse educational backgrounds and quickly - without long term commitment - bring them to a level where they can start using ML for their applications.
To address this challenge, we use the ideas borrowed from teaching of Operations Research. After providing the students with a brief overview of ML approaches and tools, we offer them a menu of possible projects that can be tweaked to their specific goals and requirements. Among such projects are the ones based on diet, assignment, scheduling and logistic problems.  Those classical problems can be tweaked to real life applications. Having selected the projects, the students continue to work on them for the rest of the course.
Our methodology may be useful for instructors of ML in professional education, as well as engineering undergraduate programs.

nadiakap commented:
> @nadiakap we're getting pretty close to the deadline here. How are the updates coming along?

 the updated abstract:

Use of ML has become mainstream in many areas of business. There is no shortage of courses offering different paths to mastering ML skills. The most well-known approaches are those of Andrew Ng and Jeremy Howard. Andrew Ng's approach assumes a decent mathematical background and is essentially deductive, while Jeremy Howard's approach is more inductive and requires high school level of math and limited coding skills. Obviously, both are valuable in their own ways.
In the practice of continual education, however, the unique challenge is to work with people of diverse educational backgrounds and quickly - without long term commitment - bring them to a level where they can start using ML for their applications.
To address this challenge, we use the ideas borrowed from teaching of Operations Research. After providing the students with a brief overview of ML approaches and tools, we offer them a menu of possible projects that can be tweaked to their specific goals and requirements. Among such projects are the ones based on diet, assignment, scheduling and logistic problems.  Those classical problems can be tweaked to real life applications. Having selected the projects, the students continue to work on them for the rest of the course.
Our methodology may be useful for instructors of ML in professional education, as well as engineering undergraduate programs.

nadiakap commented:
the updated abstract :
Use of ML has become mainstream in many areas of business. There is no shortage of courses offering different paths to mastering ML skills. The most well-known approaches are those of Andrew Ng and Jeremy Howard. Andrew Ng's approach assumes a decent mathematical background and is essentially deductive, while Jeremy Howard's approach is more inductive and requires high school level of math and limited coding skills. Obviously, both are valuable in their own ways.
In the practice of continual education, however, the unique challenge is to work with people of diverse educational backgrounds and quickly - without long term commitment - bring them to a level where they can start using ML for their applications.
To address this challenge, we use the ideas borrowed from teaching of Operations Research. After providing the students with a brief overview of ML approaches and tools, we offer them a menu of possible projects that can be tweaked to their specific goals and requirements. Among such projects are the ones based on diet, assignment, scheduling and logistic problems.  Those classical problems can be tweaked to real life applications. Having selected the projects, the students continue to work on them for the rest of the course.
Our methodology may be useful for instructors of ML in professional education, as well as engineering undergraduate programs.

nadiakap commented:
> the updated abstract : Use of ML has become mainstream in many areas of business. There is no shortage of courses offering different paths to mastering ML skills. The most well-known approaches are those of Andrew Ng and Jeremy Howard. Andrew Ng's approach assumes a decent mathematical background and is essentially deductive, while Jeremy Howard's approach is more inductive and requires high school level of math and limited coding skills. Obviously, both are valuable in their own ways. In the practice of continual education, however, the unique challenge is to work with people of diverse educational backgrounds and quickly - without long term commitment - bring them to a level where they can start using ML for their applications. To address this challenge, we use the ideas borrowed from teaching of Operations Research. After providing the students with a brief overview of ML approaches and tools, we offer them a menu of possible projects that can be tweaked to their specific goals and requirements. Among such projects are the ones based on diet, assignment, scheduling and logistic problems. Those classical problems can be tweaked to real life applications. Having selected the projects, the students continue to work on them for the rest of the course. Our methodology may be useful for instructors of ML in professional education, as well as engineering undergraduate program 

@deniederhut ,  I will be able to submit the corrected paper on July 17

nadiakap committed:
final version

all files updated

nadiakap commented:
final version is submitted

deniederhut commented:
@scoobies build status

scoobies commented:
{"nadiakap-2023"=>{"data"=>{"build_output"=>"[*] Updating proceedings build tools in {self.master_repo_path}...\nAlready up to date.\n[*] Check out paper repository...\nCloning into '/tmp/tmpaliai_sl'...\nMoving proceedings build tools to temp directory...\n[*] Build the paper...\nBuilding: nu\nTraceback (most recent call last):\n  File \"publisher/build_paper.py\", line 681, in <module>\n    build_paper(paper_id)\n  File \"publisher/build_paper.py\", line 664, in build_paper\n    metadata_path = prepare_metadata(out_path)\n  File \"publisher/build_paper.py\", line 336, in prepare_metadata\n    metadata = preprocess_metadata(metadata)\n  File \"publisher/build_paper.py\", line 316, in preprocess_metadata\n    new['abstract'] = [s.strip() for s in abstract]\n  File \"publisher/build_paper.py\", line 316, in <listcomp>\n    new['abstract'] = [s.strip() for s in abstract]\nAttributeError: 'dict' object has no attribute 'strip'\nError building paper /tmp/tmpaliai_sl/papers/nu. Aborting.\n[X] Error code 1\nA build error occurred during the self.run_make_paper_script step.", "build_pdf_path"=>"", "build_status"=>"Build failed, make_paper.sh did not succeed", "build_timestamp"=>"18/07 00:16"}, "status"=>"fail"}}

deniederhut commented:
@nadiakap  it can't be final if it doesn't build

nadiakap committed:
fix to abstract

nadiakap committed:
update paper

nadiakap commented:
> @nadiakap it can't be final if it doesn't build

corrections submitted

nadiakap commented:
> > @nadiakap it can't be final if it doesn't build
> 
> corrections submitted

@deniederhut would you please send me log if there is any issues.

deniederhut commented:
@nadiakap you can request the build logs by using the command that I used here https://github.com/scipy-conference/scipy_proceedings/pull/818#issuecomment-1639086142

deniederhut commented:
You have created your abstract as a YAML array. If you want a multiline abstract there is different syntax for that https://stackoverflow.com/a/21699210

deniederhut commented:
@nadiakap did you present this poster at the conference?

nadiakap commented:
> 

no I did not. I could not go there, and virtual poster I could not submit to the new system. I have it as .pdf in my ML class repo

nadiakap commented:
> You have created your abstract as a YAML array. If you want a multiline abstract there is different syntax for that https://stackoverflow.com/a/21699210

I wanted to keep the same format as in previous version of YAML. Only added couple of sections. not sure what multiline means

deniederhut commented:
I'm sorry to hear that. Virtual poster presenters are still expected to register for the conference and attend -- there was a remote option available for anyone who could not attend in person. 

We can't accept a paper into the proceedings that did not appear at the conference.

deniederhut commented:
@scoobies mark not ready

nadiakap commented:
> @scoobies mark not ready

oh ok, i see

nadiakap commented:
do I have to withdraw my submission somehow?

nadiakap commented:
> do I have to withdraw my submission sunless I can register now.. but I am afraid it is too late..
